examples/download-progress/tui.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"strings"
+	"time"
+
+	"github.com/charmbracelet/bubbles/progress"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+var helpStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("#626262")).Render
+
+const (
+	padding  = 2
+	maxWidth = 80
+)
+
+type progressMsg float64
+
+type progressErrMsg struct{ err error }
+
+type finalPauseFinishedMsg struct{}
```

Comment:
I _think_ we can drop this one: I believe it’s residue from an earlier iteration.